### PR TITLE
Update Advanced Analytics config guides

### PR DESCRIPTION
**Pull request by ajosh0504**
## Issues
[ia-trade-team/206](https://github.com/elastic/ia-trade-team/issues/206)

## Summary
This PR adds more detailed configuration guides to rules associated with the following integration packages:
- dga
- problemchild
- ded
- lmd
- beaconing

### CODE REVIEW

**Comment by terrancedejesus** [comment]
@ajosh0504 - `event.ingested = "timestamp"` is enforced in the Detection Rules repository as we consider it the source of truth for the event regarding timestamps. This is only enforced if the rule type is not `eql` or `query`.

However since your analytic packages have rules that are of rule type `query`, we now have an exception that must be accounted for. In this instance, we have already defined what an analytic package is in definitions.py, so we just have to cross reference this for the unit test and it should pass. 

I'll push a commit shortly and then we can move forward with your PR.

**Comment by terrancedejesus** [comment]
@ajosh0504 - [commit](https://github.com/elastic/detection-rules/pull/3302/commits/2fa82da815d817cc3a107e2dae193bdf486e3961) added to fix this. Should be good to go, but let me know if you have any questions.

File: rules/integrations/beaconing/command_and_control_beaconing.toml [hunk 1]
```diff
@@ -18,20 +18,35 @@ index = ["ml_beaconing.all"]
 language = "kuery"
 license = "Elastic License v2"
 name = "Statistical Model Detected C2 Beaconing Activity"
-note = """## Setup
+setup = """
+
```
[the remark this answers is not in the document]
Comment (ajosh0504) [reply rules/integrations/beaconing/command_and_control_beaconing.toml]: Removed them!

**Comment by terrancedejesus** [comment]
@ajosh0504 @approksiu - While we have this PR open, do we want to add `endpoint` and `network_traffic` integration tags to these rules? Kseniia had mentioned they are required for these?

File: tests/test_all_rules.py [hunk 2]
```diff
@@ -606,6 +606,7 @@ def test_integration_tag(self):
         """Test integration rules defined by metadata tag."""
         failures = []
         non_dataset_packages = definitions.NON_DATASET_PACKAGES + ["winlog"]
+        analytic_packages = [*map(str.lower, definitions.MACHINE_LEARNING_PACKAGES)]
```
Comment (terrancedejesus) [inline tests/test_all_rules.py:609]:
```suggestion
```

**Review by Mikaayenson — APPROVED** [review]
LGTM

**Review by approksiu — APPROVED** [review]
Thanks for this! LGTM!

**Pull request merged**